tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react"
```

Comment:
**nitpick**: why it is required to change?

Reply:
I reverted this change, is not necessary,

Reply:
Reverted https://github.com/wazuh/wazuh-kibana-app/pull/5166/commits/e0b15e4cc22d0177b3ee4edb598c527c37f4e540